### ISSUE TITLE
Revision -- Feedback 2

### ISSUE THREAD

**Issue by apoorvagnihotri** [distillpub/post--bayesian-optimization#5]
- [x] Add citation for Active learning. Burr Settles has a good review. There is another highly cited survey paper. Cite both. The citation should be added when we mention Problem 1.
- [x] When you introduce Gaussian Processes, the citations seem to be broken.
- [x] Footnote on Matern kernel comes after the end of the line. It should come before “.”
- [x] When you talk about ground truth for the first time, denote it with a function “f(x)”
- [x] In the figure showing the prior - label the Predicted as Predicted (mu), GT as GT (f). Do the same for all the plots. This will ensure that everyone is clear about f, mu, sigma, GT and predicted.
- [x] f(x^+) refers to the maximum functional value, i.e., max(train_y) — you have already introduced some convention, no need to use train_y, just use the convention introduced earlier.
- [x] Correct the figure showing PI and CDF: make initial epsilon 0.0 instead of 0; 
  - [x] the font size are not uniform compared to earlier plots, these seem to be smaller in this plot, 
  - [x] try to put the PI on the log-scale so that we can hopefully show that increasing epsilon leads to a different point being chosen by PI.
- [x] Consistency for hyper-parmater v/s hyperparameter. Either is okay — just use the same one everywhere.
- [x] Reduce usage of code Python block. For example, for “how likely” in EI, either make it strong or italic, but no need to put it in code block. Correct others too.
- [x] Convention : EI has a different meaning for x* as compared to x. Maybe in PI also correct it and make it x_{t++1}
- [x] Mockus citation does not show properly from the main text.
- [x] In the equation after talking about Mockus, you seem to have omitted _{t+1}. Let’s ensure the correctness and consistency across the entire post.
- [x] mean(x) - f(x^+) : Use the introduced symbols for mean(x)? And similarly for the next one also write sigma
- [x] In the PI v.s. EI graph — write epsilon_{PI} = 0.01 and similarly for epsilon_{EI}.
- [x] In the GP-UCB formulation, mention what is small t.
- [x] In GP-UCB mention how it looks very similar to UCB introduced earlier, but differs in the fact that the second term has a time-dependent/iteration-dependent coefficient. 
- [x] Remove the discussion about Bernoulli
- [x] For EI-PI mention all the hyperparams for both the PI and the EI part of it. Also, mention that such combinations can help overcome the limitations of individual methods. 
- [x] In the comparison plot, x-axis should be number of drilled sites
- [x] In the comparison plot, mention that the random experiment was done with multiple random seeds and we show the mean and the variance. Mention that the variance is very high. 
- [x] Mention that for all the other methods, we took the “best” hyper parameters for each approach.
- [x] One small example that we can think of can be of linear regression, we don't really have hyperparameters, but the parameters are the  — replace this discussion with Ridge regression (write the objective, mention \theta being a param, mention \lambda >0 and learning rate to be hyperparams.
- [x] Be consistent with let’s or let us. Just prefer the latter as it more formal. 
- [x] For the sklearn moon dataset, add legend for Class A and Class B.
- [x] In SVM section, mention briefly what C and gamma do. 
- [x] The font size in the SVM plot is not consistent with the remaining figures and is very small. 
- [x] In SVM plot, the title should be accuracy and accuracies.
- [x] In SVM plot, the accuracy should be on a scale of 40 to 100 and not on 0.4 to 1. This is just for avoiding any confusion. 
- [x] In SVM plot, for both the subplots, take the legend outside the main plot. Else, it is very hard to see what is happening. 
- [x] In SVM plot, the global maxima is hard to see. Make it much bigger. 
- [x] They with different CMAPS — https://matplotlib.org/users/colormaps.html  Viridis might be okay.
- [x] Comparison between methods should be renamed to comparison of different acquisition functions on SVM classification task.
  - [x] font size should be made consistent with all plots
  - [x] make the lines a little thicker, 
  - [x] GP_UCB in legend should be GP-UCB
  - [x] the X-label should be # queries made
- [x] Random Forest — isn’t our task classification, you seem to mention using RF regressor. As before with SVM one, take the legend out. Add a bit of explanation for depth v/s number of trees. 
- [x] The plot looks a little odd. Maybe it is due to the legend intermingling. 
- [x] On RFs, the comparison plot for random doesn’t seem to make sense. We can never decrease the max. Accuracy reached till now.
- [x] For example on CNN, mention what Hyperparameter space we are looking at: like batch size from 2^3 to .., similarly for others and then mention how we can easily do this in modern package and here you show how it translates to simple Python code in scikit-optimize. As before mention how changing the hyperparams might change the prediction.
- [x] The CNN example seems to end very abruptly. Mention the key learning if any. The plot also needs to be interpreted. 
- [x] In conclusions, mention that optimizing  or tuning hyperparams is an important facet of modern machine learning algorithms and BO is an efficient way of tuning the same.  I would suggest that the last line should be — we hope you had a good time reading the article and hope you are ready to <strong>exploit </strong> the power of BO. In case you wish to <strong> explore </strong> more, please read the Further reading section below.
- [x] Wherever possible don’t mention - I have. .. instead write we
- [x] I would suggest we write “Further reading” and break it down into various aspects - like a) making use of gradient information when available; b) very quickly mention or link the difference between BO and GD; c) mention the caution about kernel; d) BO applications other than the ones discussed in this paper. This should also include the sensor placement paper we saw a few days back.
- [x] some practical tips! Scaled Accuracy
